frontend/app/controllers/Info.scala

```diff
@@ -1,15 +1,21 @@
 package controllers
 
+import configuration.Config
 import configuration.CopyConfig
 import forms.MemberForm._
 import model.{FlashMessage, PageInfo}
 import play.api.mvc.Controller
 import services.EmailService
+import actions.Functions._
 
 import scala.concurrent.Future
 
 trait Info extends Controller {
 
+  val permanentStaffGroups = Config.staffAuthorisedEmailGroups
+  val AuthorisedStaff = GoogleAuthenticatedStaffAction andThen isInAuthorisedGroupGoogleAuthReq(
```

Comment:
I'm duplicating this same authentication logic from the `Offer` controller, this little awkward. Is there a way I can clean this up? //cc @jennysivapalan